pdns/dnspacket.hh

```diff
@@ -58,7 +58,7 @@ public:
 
   int noparse(const char *mesg, size_t len); //!< just suck the data inward
   int parse(const char *mesg, size_t len); //!< parse a raw UDP or TCP packet and suck the data inward
-  const string& getString(bool throwsOnTruncation=false); //!< for serialization - just passes the whole packet. If throwsOnTruncation is set, an exception will be raised if the records are too large to fit inside a single DNS payload, instead of setting the TC bit
+  const string& getString(bool leaveOverflow=false); //!< for serialization - just passes the whole packet. If leaveOverflow is set, records not fitting into a single DNS packet will be left in d_leftRRs and TC will not be set.
```

Comment:
We might need both options:
- throw if the records do not fit into a TCP packet at all, for non-XFR case
- keep non-fitting records around, for XFR cases
?